### DISCUSSION TITLE
How to use readme at root of repository as starting point of docs

### DISCUSSION THREAD

**Discussion by ppipada** [mkdocs/mkdocs#3221] [Q&A] [answered]
My repo ad docs layout is :
```
git-repo/
      - mkdocs.yml
      - README.md
      - service/
             - README.md
```
I am trying to serve this via mkdocs and not able to get it to work. 
If I specify docs_dir as "/" or ".", it wont allow. 
The issue: https://github.com/mkdocs/mkdocs/issues/2831 mentions to use pysnippets, ut even in that case, I would need to have a docs directory created and a index.md file created. 

Is there any way to do this without creating any specialized docs directory?

**Comment by rmartine-ias** [comment, marked as answer]
You can use this plugin:
https://github.com/oprypin/mkdocs-same-dir

**Reply by ppipada** [reply]
Thanks
